src/utils/colonyActions.ts

```diff
@@ -231,8 +231,12 @@ export const getSafeTransactionActionType = (
     if ((safeTransactions || []).length >= 2) {
       return TransactionTypes.MULTIPLE_TRANSACTIONS;
     }
-    const type = safeTransactions[0].transactionType;
-    return type;
+
+    if (safeTransactions.length === 1) {
```

Comment:
To guard against `safeTransactions[0]` being undefined (I can't off the top of my head think of why that would be the case; wouldn't there always be at least one tx?), could do `const type = safeTransactions[0]?.transactionType` instead?

Reply:
I was having problems with the subgraph because the annotations were not being indexed fast enough (That's why you saw that the title in the screenshot is not formatted), but I think I've solved it so this can be reverted to how it was before

Reply:
For the record, I tried with `?` and it still wasn't working due to the `[0]` if I remember correctly